assignment6/solve.py

```diff
@@ -0,0 +1,116 @@
+import unittest
+
+
+def solve(input_array, result_array):

```

Comment:
Maybe in this case it's not very relevant as we all know what kind of task we're solving, but it would seem more natural to name a function related to cars rearrangement (or array rearrangement) rather than just solve
So that later you could import it in another file and not get confused